classes/Path.php

```diff
@@ -41,11 +41,16 @@ public function configPath(): string
         return $this->path . "/config/{$this->env}.yml";
     }
 
-    public function uploadPath(): string
+    public function uploadToPath(): string
     {
         return $this->path . '/web/uploads';
     }
 
+    public function downloadFromPath(): string
```

Comment:
I am not entirely sure about the naming.

As far as I understand, both `uploadToPath()` and `downloadFromPath()` refer to the same location, with the difference that the former represents the path in the file system, and the latter represents the path relative to the web root, is that right?

What do you think?

Reply:
The upload to path is the path in the filesystem, used to save the profile image.
The download from function is the path relative from the web root, correct.

I agree that the naming might be a bit confusing, so if you have any ideas i'd be glad to change them.

But if we take for example, the uploading of images using a google cloud bucket this makes a bit more sense.
We'd use the ```gs://``` streamwrapper to upload the images, since that is done on the server itself.
But when we are showing the images to the users, that is done from their browser, so link to the images from ```https://storage.googleapis.com/```.

Reply:
@BackEndTea 

About your example, I guess it would be best to abstract the *actual* location of an image away, that is, an image could be an object that has a URL, and then none of us have to care where that image is located, or whether it needs prefixing with anything.

Maybe it would be better to move the part that deals with the download path to the `SpeakerProfile` for now? What do you think?

Reply:
I wouldn't mind doing that, but a lot of places referring to the image location do not use the ```SpeakerProfile```.